### DISCUSSION TITLE
How to open a new window using the command line

### DISCUSSION THREAD

**Discussion by Achrou** [dbeaver/dbeaver#34787] [Q&A] [answered]
`/Applications/DBeaver.app/Contents/MacOS/dbeaver -con 'create=false|id=mysql8-18874e8b754-3024ed613296f29b' -f /Users/achrou/Library/DBeaverData/workspace6/General/Scripts/Script-1.sql -reuseWorkspace`
 I can only open a new tab instead of a new window with this command

**Comment by E1izabeth** [comment, marked as answer]
You can do it with `-newInstance` option
Here is the wiki about available options https://dbeaver.com/docs/dbeaver/Command-Line/
